sdk/loadtesting/playwright/src/core/playwrightService.ts

```diff
@@ -63,7 +63,7 @@ const performOneTimeOperation = (options?: PlaywrightServiceAdditionalOptions):
  * import playwrightConfig from "./playwright.config";
  *
  * export default defineConfig(playwrightConfig, createAzurePlaywrightConfig(playwrightConfig, {
- *  runId: "custom run id",
+ *  runId: "custom run id",  // run ID should be a GUID
```

Comment:
lets remove runId from example, instead auth type and credential seems more important